epoxy-processor/src/main/java/com/airbnb/epoxy/ViewAttributeInfo.kt

```diff
@@ -226,27 +261,34 @@ internal class ViewAttributeInfo(
             errorLogger: ErrorLogger,
             options: Set<Option>,
             types: Types,
-            elements: Elements
+            elements: Elements,
+            setterElementOnView: Element
     ) {
+
+        if (!options.isEmpty() &&  viewAttributeTypeName == ViewAttributeType.Field) {
```

Comment:
a field should be able to support options, I don't think it needs to be different there